src/amo/reducers/addonsByAuthors.js

```diff
@@ -37,7 +41,7 @@ export const LOAD_ADDONS_BY_AUTHORS: 'LOAD_ADDONS_BY_AUTHORS'
   = 'LOAD_ADDONS_BY_AUTHORS';
 
 type FetchAddonsByAuthorsParams = {|
-  addonType: string,
+  addonType?: string,
   authors: Array<string>,
```

Comment:
I noticed in the `MoreAddonsByAuthorsCard` component that the array of author names is consistently referred to as `authorNames`, but here it's called `authors` (and this method is dispatched from `MoreAddonsByAuthorsCard`). For consistency it might make sense to change this to `authorNames`, plus it _is_ author names, and not author objects, so that's another reason it might be good to change it.